tests/php/test-fieldmanager-context-term.php

```diff
@@ -110,9 +122,13 @@ private function _get_html_for( $field, $test_data = null ) {
 	}
 
 	public function test_context_render_add_form() {
+		if ( ! _fm_phpunit_is_wp_at_least( 4.4 ) ) {
+			return $this->_skip_tests_because_version( 4.4 );
```

Comment:
Why not test the Fieldmanager interfaces regardless of WordPress version?

I'd think you'd want to test that term meta works for both WordPress 4.3 and WordPress 4.4